main/hgrep.hs

```diff
@@ -25,12 +28,30 @@ main = do
       IO.hPutStrLn IO.stderr er
       exitWith (ExitFailure 2)
     Right q -> do
+      -- if no directory provided -> search in current directory
+      files <- case cmdFiles opts of
+                 [] -> pure <$> D.getCurrentDirectory
```

Comment:
Nitpick: can this be indented differently (i.e less)? The rest of the file is consistently two-space indents.